webvtt/parsers.py

```diff
@@ -193,6 +194,9 @@ def _parse(self, lines):
         self._compute_blocks(lines)
 
         for block in self.blocks:
+            # skip empty blocks
+            if self._is_empty(block):
```

Comment:
I don't think we need to check if a block is empty as we only create blocks with content.

Reply:
`test_can_parse_empty_lines_from_netflix_subtitles` is a test that was failing without this check in place.   It seemed like the file netflix generated had a line with a bunch of spaces on it `"                    "` and so we ended up with a block with nothing inside it